lib/active_merchant/billing/gateways/checkout_v2.rb

```diff
@@ -215,7 +215,7 @@ def parse(body)
       end
 
       def success_from(response)
-        response['response_summary'] == 'Approved' || response.key?('action_id')
+        response['response_summary'] == 'Approved' || !response.key?('response_summary') && response.key?('action_id')
```

Comment:
We should use `()` here to make it clear the precedence of these conditions vs. having to rely solely on our knowledge of Ruby's rules around `||` and `&&`?

(obviously something to change once we've resolved any immediate issues)

Reply:
I don't think it's just ruby. I think the general order of precedence for logic operators is ~, &, ||. But `()` would make it more clear.